packages/Textarea/src/Textarea.js

```diff
@@ -27,6 +27,12 @@ const Textarea = React.forwardRef((props, ref) => {
     ...moreProps
   } = props;
 
+  React.useImperativeHandle(ref, () => ({
+    focus: () => {
+      refTextarea.current.focus();
+    },
```

Comment:
👍 I think this is a good change.  

Our other core form components (`<Input>` + `<Select>`) don't have this, they still pass the `ref` to the underlying DOM element, but I think we should probably get them all in sync in a later PR.